hs_access_control/migrations/0018_auto_tune_provenance.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does this need to be a separate migration file? I think regeneration of migration files should merge the above 2 files to one.